CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
CMake has a way to specify stringly-typed variables with multiple allowed values, see line 16.